core/src/main/java/org/infinispan/interceptors/impl/InvocationContextInterceptor.java

```diff
@@ -90,7 +86,7 @@ private Object handleAll(InvocationContext ctx, VisitableCommand command) throws
          ComponentStatus status = componentRegistry.getStatus();
          if (command.ignoreCommandOnStatus(status)) {
             log.debugf("Status: %s : Ignoring %s command", status, command);
-            return null;
+            return ctx.continueInvocation();
```

Comment:
It's another one of those methods where the return value is `Object`, so `return null` would have been better, but it was still ignoring the command.